src/screens/Main.tsx

```diff
@@ -27,19 +36,23 @@ import Remark from '../components/Remark';
 import SnackBar from '../components/SnackBar';
 import Transactions from '../components/Transactions';
 import Transfer from '../components/Transfer';
-import { useSourceTarget } from '../contexts/SourceTargetContextProvider';
 import { ChainDetails } from '../types/sourceTargetTypes';
 
 interface Props {
   className?: string;
 }
 
 function Main({ className }: Props) {
-  const { sourceChainDetails, targetChainDetails } = useSourceTarget();
-
   return (
     <BoxMain>
-      <BoxSidebar>{`${sourceChainDetails.sourceChain} => ${targetChainDetails.targetChain}`}</BoxSidebar>
+      <BoxSidebar>
+        <div>
```

Comment:
is this div needed at all?

Reply:
for justify space-between layout for direct children of BoxSidebar

Reply:
I'd keep it like this. Div stands for division, so it makes sense even semantically to create this separation with divs, rather than with margins. For future version there might be menu items, also in divs